src/materials/MeshPhongMaterial.js

```diff
@@ -94,6 +97,9 @@ function MeshPhongMaterial( parameters ) {
 	this.reflectivity = 1;
 	this.refractionRatio = 0.98;
 
+	this.toonMap = null;
```

Comment:
Yes.

With like the toonImage I put above
it can make object look like flat=toon-style.